<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageWidget</class>
 <widget class="QWidget" name="MessageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MessageListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>396</width>
     <height>308</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>394</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="SuspendedScrollBar_TextEdit" name="edit_input">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>350</y>
     <width>344</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_emoji">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>319</y>
     <width>25</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_voice_chat">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>319</y>
     <width>25</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_video_chat">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>319</y>
     <width>25</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageListWidget</class>
   <extends>QListWidget</extends>
   <header>MessageItem.h</header>
  </customwidget>
  <customwidget>
   <class>SuspendedScrollBar_TextEdit</class>
   <extends>QTextEdit</extends>
   <header>SuspendedScrollBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
